internal/convertor/snmp/snmp.go

```diff
@@ -0,0 +1,39 @@
+package snmp
+
+import (
+	"github.com/criteo/data-aggregation-api/internal/model/cmdb/snmp"
+	"github.com/criteo/data-aggregation-api/internal/model/ietf"
+)
+
+func SNMPtoIETFfSystem(snmp *snmp.SNMP) ietf.IETFSystem_System {
```

Comment:
as SNMP is optional, you need to check if snmp is nil as it is a pointer

like done for BGP Global (in global.go):
```
	if bgpGlobal == nil {
		return nil
	}
```